blocks/api/test/serializer.js

```diff
@@ -195,13 +199,14 @@ describe( 'block serializer', () => {
 				{
 					name: 'core/test-block',
 					attributes: {
+						foo: false,
```

Comment:
@aduth thanks for the feedback. Done in 71a07fc9bac6348a28545ad9cfe23e91c6c50a26.